mono/metadata/custom-attrs.c

```diff
@@ -980,15 +980,17 @@ create_custom_attr (MonoImage *image, MonoMethod *method, const guchar *data, gu
 			mono_property_set_value_handle (prop, attr, pparams, error);
 			goto_if_nok (error, fail);
 		}
+		
+		g_free (name);
```

Comment:
Shouldn't we NULL-out the value after it's been freed?

```suggestion
		g_free (name);
		name = NULL;
```